frontend/src/components/participant_profile/profile_preview.ts

```diff
@@ -40,6 +47,16 @@ export class Preview extends MobxLitElement {
       return nothing;
     }
 
+    this.participantService.updateForRoute(
```

Comment:
Can you clarify what this is doing (and why it's called twice)?

Reply:
Yes! It's not called twice; we update the participantService and the participantAnswerService. We want to fetch the current survey answer data, which relies on participantAnswerService (which in turn relies on participantService), so the routes need to be updated. If this is not included, participantAnswerService will not be able to provide the current survey answer data.

Reply:
Ah sorry, I missed the `answer` in `participantAnswerService`!

The answer service is used for saving participants' local edits, so I'm not sure it makes sense to be using it in the profile preview (as experimenters or anyone else on this page would only be viewing what's saved in Firebase). Can we load information from participantService (which should have an answer map) only?

Reply:
Update: sorry, I just realized you're using the answer service because the stage view components rely on it. Let me think about this some more.

Reply:
What do you think about updating these when the experimenter clicks on the current participant (rather than in the lit component)? That way, the participant answers are up-to-date on the frontend (based on the experiment manager's currently viewed participant) regardless of whether or not this component is shown.

Reply:
Sounds fine to me (esp. given the true purpose of the answer service); would you mind making this change? I'm not sure I follow entirely on the implementation.

Reply:
Yes, I'll add it to this PR before merging!

Reply:
FYI, this is the fix I pushed: https://github.com/PAIR-code/deliberate-lab/pull/310/commits/de96f3aeb50871a1e108f89ea5a2f5b6cb90389a